generate-stackbrew-library.sh

```diff
@@ -48,13 +54,13 @@ getArches() {
 	local officialImagesUrl='https://github.com/docker-library/official-images/raw/master/library/'
 
 	eval "declare -g -A parentRepoToArches=( $(
-		find -name 'Dockerfile' -exec awk '
-				toupper($1) == "FROM" && $2 !~ /^('"$repo"'|scratch|.*\/.*)(:|$)/ {
+		find -name Dockerfile -exec awk '
+			/^FROM/ {
+				if (index($2, ":") > 0 && index($2, "'"$repo"'") == 0) {
```

Comment:
Long-term we might want to rewrite this to use a hash lookup table to track `AS` values so we can ignore them in subsequent `FROM` instructions, but this should be fine for now (technically, we can also have remote images / remote dependencies in `COPY --from=`, but I don't see a strong reason for us to use those here yet so we can cross that entire bridge if/when we come to it). :+1: